README.md

```diff
@@ -306,6 +306,510 @@ val byteToHex = FunSpec.builder("byteToHex")
     .build()
 ```
 
+### Functions
+
+All of the above functions have a code body. Use `KModifier.ABSTRACT` to get a function without any
+body. This is only legal if the enclosing class is either abstract or an interface.
```

Comment:
only legal if it is enclosed by an abstract class or interface?

otherwise we imply top-level functions don’t exist